Jenkinsfile

```diff
@@ -19,8 +19,9 @@ pipeline {
 
     // Only publish to RubyGems if branch is 'master'
     // AND someone confirms this stage within 5 minutes
-    stage('Publish to RubyGems?') {
 
+    stage('Publish to RubyGems') {
+/*
```

Comment:
coment?

Reply:
I forgot to uncomment this after testing. It's the when clause to make it only publish when a release is needed.